### ISSUE TITLE
Add the possibility of defining aliases for columns

### ISSUE THREAD

**Issue by mchojrin** [petkopara/PetkoparaCrudGeneratorBundle#36]
I'm seeing a column name in my index.html that I'd like to change for something more readable (view wise).

I believe it could be achieved by changing the parameters in vendor/petkopara/crud-generator-bundle/Resources/views/macros/th_sortable.html.twig:

If colName whas actually (perhaps optionally) an array it should be easy to do.

**Comment by RodolVelasco** [comment]
@mchojrin did you find a way to change the name of a column?

**Comment by mchojrin** [comment]
@RodolVelasco not really. I have to say though I haven't tried that hard either... I've been busy with other things. I guess the way to go would be to try that on a sandbox project and, if it works, create a pull request to submit the improvement...
